environs/manual/provisioner.go

```diff
@@ -50,9 +51,22 @@ type ProvisionMachineArgs struct {
 	// ubuntu user's ~/.ssh/authorized_keys.
 	AuthorizedKeys string
 
+	// WKeys winrm keys that contains, CACert, ClientCert, ClientKey
+	WKeys *winrm.X509
```

Comment:
Can you please expand the W here and below to WinRM. It's not very meaningful out of context